### PR TITLE
Feature/180588 replace existing mock with wiremock

### PR DESCRIPTION
**Pull request by rushikeshkhopatkar**
Hi,
This PR contains code for ERP Facade Wiremock. This is a temporary mock to enable development & testing of sprint 3 PBIs. Once the actual mock is ready, this shall be replaced with it.

Please review.

Thanks.

### CODE REVIEW

File: src/UKHO.SAP.MockAPIService/Program.cs [hunk 1]
```diff
@@ -1,86 +1,34 @@
...
-using UKHO.ERPFacade.Common.Operations.IO.Azure;
-using UKHO.SAP.MockAPIService.Filters;
-using UKHO.SAP.MockAPIService.Services;
+﻿using UKHO.ERPFacade.StubService.Configuration;
+using UKHO.ERPFacade.StubService.StubSetup;
+using UKHO.SAP.MockAPIService.Configuration;
+using WireMock.Settings;
 
-namespace UKHO.SAP.MockAPIService
+namespace UKHO.ERPFacade.StubService
```
Comment (Vishal14583) [inline src/UKHO.SAP.MockAPIService/Program.cs:6]: Please correct the namespace.

File: src/UKHO.SAP.MockAPIService/Configuration/RecordOfSaleEventConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace UKHO.ERPFacade.StubService.Configuration
```
Comment (Vishal14583) [inline src/UKHO.SAP.MockAPIService/Configuration/RecordOfSaleEventConfiguration.cs:1]: Please correct the namespace

File: src/UKHO.SAP.MockAPIService/Configuration/S100DataEventConfiguration.cs [hunk 3]
```diff
@@ -0,0 +1,7 @@
+﻿namespace UKHO.ERPFacade.StubService.Configuration
```
Comment (Vishal14583) [inline src/UKHO.SAP.MockAPIService/Configuration/S100DataEventConfiguration.cs:1]: Please correct the namespace

File: src/UKHO.SAP.MockAPIService/Configuration/S57EncEventConfiguration.cs [hunk 4]
```diff
@@ -0,0 +1,7 @@
+﻿namespace UKHO.ERPFacade.StubService.Configuration
```
Comment (Vishal14583) [inline src/UKHO.SAP.MockAPIService/Configuration/S57EncEventConfiguration.cs:1]: Please correct the namespace wherever applicable

**Review by Vishal14583 — CHANGES_REQUESTED** [review]
Few comments around namespaces

**Review by Vishal14583 — APPROVED** [review]
LGTM

**Review by jagannathsankpal — APPROVED** [review]
LGTM

**Review by rushdynajath — APPROVED** [review]
Looks good

**Pull request merged**